heuristic/delta_calculations.py

```diff
@@ -309,15 +309,15 @@ def calc_weekly_objective_function(state, competencies, time_periods_in_week, co
                                    for t in combined_time_periods_in_week[j]
                                    )
 
-                        - max(0,
-                              sum(2 * state.soft_vars["deviation_contracted_hours"].get((e, j), 0)
-                                  for e in employees
-                                  )
-                              )
+                        # - max(0,
+                        #       sum(2 * state.soft_vars["deviation_contracted_hours"].get((e, j), 0)
+                        #           for e in employees
+                        #           )
+                        #       )
 
-                        - 10 * sum(state.hard_vars["delta_positive_contracted_hours"].get(e, 0)
-                                   for e in employees
-                                   )
+                        # - 10 * sum(state.hard_vars["delta_positive_contracted_hours"].get(e, 0)
+                        #            for e in employees
+                        #            )
```

Comment:
I believe that by not taking contracted hours into account the worst week operator is a bit better. Would like input on if this is a good idea.

Reply:
Would this affect any other operators? I think it´s hard to say wheter or not it´s a good idea, but it seems reasonable enough to warrant testing.

Reply:
All of my new test are done with this tweak. It is difficult to say if the results are because of this tweak or mostly because of the new operator

Reply:
Can you run a couple of tests with contracted hours restored? It would be nice to be able to compare with your previous results

Reply:
Will do